subsys/logging/log_output.c

```diff
@@ -43,8 +43,8 @@ static const char *const colors[] = {
 	NULL                    /* dbg */
 };
 
-static uint32_t freq;
-static uint32_t timestamp_div;
+static log_timestamp_t freq;
```

Comment:
both variables should stay uint32_t since it's frequency and frequency divider.

Reply:
Yes, of course the same holds also for this point. I'll revert it